simulations/vip-152/vip-152/simulation.ts

```diff
@@ -0,0 +1,94 @@
+
+const vBNB_ADDRESS = "0xa07c5b74c9b40447a954e1466938b865b6bbea36";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+const VBNBAdmin_ADDRESS = "0x2b11a94DA41a5cAcAa6e1E3F23139cED805808b5";
+const RISK_FUND = "0xdF31a28D68A2AB381D42b380649Ead7ae2A76E42";
+const TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+const PSR = "0x4E5A49Ce81993504327a848167d76212b7a341E2";
+
+forking(32374397, () => {
+  const provider = ethers.provider;
+
+  describe("Pre-VIP behavior", async () => {
+    let vBNB: ethers.Contract;
+    let psr: ethers.Contract;
+
+    before(async () => {
+      vBNB = new ethers.Contract(vBNB_ADDRESS, vBNB_ABI, provider);
+      psr = new ethers.Contract(PSR, PSR_ABI, provider);
+    });
+
+    it("validate admin", async () => {
+      expect(await vBNB.admin()).to.be.equal(NORMAL_TIMELOCK);
+    });
+
+    it("config check", async () => {
+      expect(await psr.totalDistributions()).to.be.equal(0);
+    });
+
+    it("ownership check", async () => {
+      expect(await psr.pendingOwner()).to.be.equal(NORMAL_TIMELOCK);
+    });
+  });
+
+  testVip("VIP-152", vip152());
+
+  describe("Post-VIP behavior", async () => {
+    let vBNB: ethers.Contract;
+    let psr: ethers.Contract;
+    let vBNBAdmin: ethers.Contract;
+
+    before(async () => {
+      vBNB = new ethers.Contract(vBNB_ADDRESS, vBNB_ABI, provider);
+      psr = new ethers.Contract(PSR, PSR_ABI, provider);
+      vBNBAdmin = new ethers.Contract(VBNBAdmin_ADDRESS, vBNBAdmin_ABI, provider);
+    });
+
+    it("validate admin", async () => {
+      expect(await vBNB.admin()).to.be.equal(VBNBAdmin_ADDRESS);
+    });
+
+    it("validate PSR", async () => {
+      expect(await vBNBAdmin.protocolShareReserve()).to.be.equal(PSR);
+    });
+
+    it("config check", async () => {
+      expect(await psr.totalDistributions()).to.be.equal(4);
+    });
+
+    it("ownership check", async () => {
+      expect(await psr.pendingOwner()).to.be.equal("0x0000000000000000000000000000000000000000");
+      expect(await psr.owner()).to.be.equal(NORMAL_TIMELOCK);
+    });
+
+    it("configurations", async () => {
+      let config = await psr.distributionTargets(0);
+      expect(config.schema).to.be.equal(0);
+      expect(config.percentage).to.be.equal(50);
+      expect(config.destination).to.be.equal(RISK_FUND);
+
+      config = await psr.distributionTargets(1);
+      expect(config.schema).to.be.equal(0);
+      expect(config.percentage).to.be.equal(50);
+      expect(config.destination).to.be.equal(TREASURY);
+
+      config = await psr.distributionTargets(2);
+      expect(config.schema).to.be.equal(1);
+      expect(config.percentage).to.be.equal(40);
+      expect(config.destination).to.be.equal(RISK_FUND);
+
+      config = await psr.distributionTargets(3);
+      expect(config.schema).to.be.equal(1);
+      expect(config.percentage).to.be.equal(60);
+      expect(config.destination).to.be.equal(TREASURY);
+    });
```

Comment:
I would add a simulation of releasing funds, from the VBNB to the Treasury and Risk func, checking the amounts are the expected ones